runner full image PATH 缺 $HOME/.pub-cache/bin，dart pub global 装的工具找不到

## 现象

#248 Phase C 派 REQ-657 实测：dev_cross_check 阶段 ttpos-flutter 跑 \`make ci-setup\` 装 melos 成功，但 \`make ci-lint\` 撞：

\`\`\`
=== dev_cross_check (ci-setup): ttpos-flutter ===
=== Activating global tools ===
Resolving dependencies...
+ melos 7.5.1
Activated melos 7.5.1.

Warning: Pub installs executables into \$HOME/.pub-cache/bin, which is not on your path.

make: melos: No such file or directory
make: *** [Makefile:129: ci-setup] Error 127
/bin/sh: 12: melos: not found
make: *** [Makefile:144: ci-lint] Error 127
=== FAIL: ttpos-flutter ===
\`\`\`

## 根因

\`runner/Dockerfile\` (full flavor, \`ghcr.io/phona/sisyphus-runner:main\`) 基于 \`ghcr.io/cirruslabs/flutter:stable\`，含 Flutter SDK / dart / pub。

但 \`dart pub global activate <pkg>\` 把 executables 装到 \`\$HOME/.pub-cache/bin\`，而 runner image PATH 没含这个目录。Pub 自己 warn 了「Pub installs executables into \$HOME/.pub-cache/bin, which is not on your path」。

\`make ci-setup\` 在子 shell 跑成功（\`dart pub global activate melos\`），但 \`make ci-lint\` 后续调 \`melos\` 时 sh 找不到 → exit 127。

## 影响

- 任何 Flutter 仓的 dev_cross_check 都会撞（不限 ttpos-flutter）
- staging_test 也受影响（同样调 melos）
- 等于 sisyphus full runner 拿不到 dart pub global tools

实际状态：自从 PR #272 切 chart 默认 \`runner.image=sisyphus-runner:main\` 起，所有 Flutter REQ 实质都被这条 silently 卡。

## 修法

\`runner/Dockerfile\` (full flavor) 加：

\`\`\`dockerfile
ENV PATH="\$PATH:\$HOME/.pub-cache/bin"
# 或等价：
ENV PATH="\$PATH:/root/.pub-cache/bin"   # runner pod 默认 root 跑
\`\`\`

也建议在 Dockerfile build 时直接预装 melos 全局：

\`\`\`dockerfile
RUN dart pub global activate melos
ENV PATH="\$PATH:/root/.pub-cache/bin"
\`\`\`

这样 runner cold start 时不用 ci-setup 重新 \`dart pub global activate\` 浪费 30s，每个 Flutter REQ 直接拿到 melos。

## 不阻塞

#248 Phase C 不卡——REQ-657 准备 admin/resume action=pass for dev_cross_check 进 staging_test。但每个 Flutter REQ 都 admin resume = 长期 ROI 差。

## 来源

#248 Phase C REQ-657 dev_cross_check 实测发现。runner 切 full flavor (PR #272) 暴露的下游 PATH 问题，立 sub-issue 跟踪。

--- a/runner/Dockerfile
+++ b/runner/Dockerfile
@@ -49,8 +49,11 @@ RUN curl -fsSL https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
 ARG GO_VERSION=1.23.4
 RUN curl -fsSL https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz \
       | tar -xz -C /usr/local
-ENV PATH="/usr/local/go/bin:/root/go/bin:$PATH" \
+ENV PATH="/usr/local/go/bin:/root/go/bin:/root/.pub-cache/bin:$PATH" \
     GOPATH=/root/go
+# /root/.pub-cache/bin (#276) — `dart pub global activate <pkg>` 装的可执行文件落
+# 在这里（如 melos / flutterfire_cli），下一个 shell 不在 PATH 就 `command not
+# found`。Flutter 业务仓 melos bs 这种步骤都靠这条路径。
 
 # ─── 4. openspec CLI ─────────────────────────────────────────────────────
 # 真包名是 @fission-ai/openspec（旧 Dockerfile 装的 @fission-codes/openspec 不存在，

--- a/thanatos/src/thanatos/server.py
+++ b/thanatos/src/thanatos/server.py
@@ -35,10 +35,37 @@ def _build_server() -> Server:
                     "type": "object",
                     "required": ["skill_path", "spec_path", "scenario_id", "endpoint"],
                     "properties": {
-                        "skill_path": {"type": "string"},
-                        "spec_path": {"type": "string"},
-                        "scenario_id": {"type": "string"},
-                        "endpoint": {"type": "string"},
+                        "skill_path": {
+                            "type": "string",
+                            "description": (
+                                "Absolute path to skill.yaml (FILE, not the .thanatos/ "
+                                "directory). Example: "
+                                "/workspace/source/<repo>/.thanatos/skill.yaml"
+                            ),
+                        },
+                        "spec_path": {
+                            "type": "string",
+                            "description": (
+                                "Absolute path to a spec.md file containing the "
+                                "Scenario block. May live anywhere; common paths: "
+                                "<repo>/.thanatos/spec-fixtures/*.spec.md or "
+                                "<repo>/openspec/changes/<REQ>/specs/.../spec.md"
+                            ),
+                        },
+                        "scenario_id": {
+                            "type": "string",
+                            "description": (
+                                "Identifier in the `#### Scenario: <id> — ...` heading "
+                                "(e.g. ttpos-shop-smoke-001 / REQ-1004-S1). Must match exactly."
+                            ),
+                        },
+                        "endpoint": {
+                            "type": "string",
+                            "description": (
+                                "Driver-specific endpoint. adb: host:port (e.g. "
+                                "localhost:5555). playwright: URL. http: base URL."
+                            ),
+                        },
                     },
                 },
             ),
@@ -52,9 +79,23 @@ def _build_server() -> Server:
                     "type": "object",
                     "required": ["skill_path", "spec_path", "endpoint"],
                     "properties": {
-                        "skill_path": {"type": "string"},
-                        "spec_path": {"type": "string"},
-                        "endpoint": {"type": "string"},
+                        "skill_path": {
+                            "type": "string",
+                            "description": (
+                                "Absolute path to skill.yaml (FILE, not directory). "
+                                "Example: /workspace/source/<repo>/.thanatos/skill.yaml"
+                            ),
+                        },
+                        "spec_path": {
+                            "type": "string",
+                            "description": "Absolute path to spec.md file.",
+                        },
+                        "endpoint": {
+                            "type": "string",
+                            "description": (
+                                "Driver-specific endpoint (adb host:port / playwright URL / http base URL)."
+                            ),
+                        },
                     },
                 },
             ),
@@ -69,12 +110,28 @@ def _build_server() -> Server:
                     "type": "object",
                     "required": ["skill_path", "intent"],
                     "properties": {
-                        "skill_path": {"type": "string"},
-                        "intent": {"type": "string"},
+                        "skill_path": {
+                            "type": "string",
+                            "description": (
+                                "Absolute path to skill.yaml (FILE). recall reads its "
+                                "parent directory to find anchors.md / flows.md / pitfalls.md / etc."
+                            ),
+                        },
+                        "intent": {
+                            "type": "string",
+                            "description": (
+                                "Free-form natural language describing what knowledge to recall, "
+                                "e.g. 'login screen widgets' or 'payment flow pitfalls'."
+                            ),
+                        },
                         "limit": {"type": "integer", "default": 10},
                         "tags": {
                             "type": "array",
                             "items": {"type": "string"},
+                            "description": (
+                                "Optional YAML frontmatter tags to filter by. "
+                                "Files without matching tags are excluded."
+                            ),
                         },
                     },
                 },

